cli/main.c

```diff
@@ -905,31 +905,44 @@ static int log_dump(int argc, char **argv)
 		int out_fd;
 		const char *out_filename;
 		unsigned type;
+		FILE *log_def_file;
 	} cfg = {
-		.type = SWITCHTEC_LOG_RAM
+		.type = SWITCHTEC_LOG_RAM,
+		.out_fd = 0,
+		.log_def_file = NULL
 	};
 	const struct argconfig_options opts[] = {
 		DEVICE_OPTION,
-		{"filename", .cfg_type=CFG_FD_WR, .value_addr=&cfg.out_fd,
+		{"output_file", .cfg_type=CFG_FD_WR, .value_addr=&cfg.out_fd,
 		  .argument_type=optional_positional,
 		  .force_default="switchtec.log",
-		  .help="image file to display information for"},
+		  .help="log output file"},
+        {"log_def_file", .cfg_type=CFG_FILE_R, .value_addr=&cfg.log_def_file,
```

Comment:
I think log_def_file should be an option argument with a flag, not a positional argument.